tools/miqssh/miqhosts.rb

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env ruby
+#
+# Parse miqhosts file for use with miqssh etc.
+#
+
+# Method load_file to read the hosts file and create useful datastructures.
+def load_file(file)
```

Comment:
You probably want to check out [LinuxAdmin::Hosts](https://github.com/ManageIQ/linux_admin/blob/master/lib/linux_admin/hosts.rb)

Reply:
miqhosts has a mapping of appliance name to a csv separated list of groups.  miqhosts.rb searches miqhosts for all hosts that are in a particular group, and can list the groups.  LinuxAdmin::Hosts seems to be for manipulating the hostname and hosts files in the format of IP hostname and optional comment.  Can it handle the groups stuff?